vscode/src/telemetry.ts

```diff
@@ -48,10 +46,18 @@ type WrappedTelemetryEvent = {
   };
 };
 
-export function initTelemetry() {
+export function initTelemetry(context: vscode.ExtensionContext) {
+  const packageInfo = getPackageInfo(context);
+  if (!packageInfo) {
+    return;
+  }
   // see issue here: https://github.com/microsoft/vscode-extension-telemetry/issues/183
   // we cannot use the latest version of extension-telemetry until this is fixed
-  const reporter = new TelemetryReporter("qsharp-vscode", "0.0.0", key);
+  const reporter = new TelemetryReporter(
+    "qsharp-vscode",
+    "0.0.0",
```

Comment:
What is the "0.0.0" here for? Looking at the constructor, this doesn't look like a 'ReplacementOptions[]' (https://github.com/microsoft/vscode-extension-telemetry/blob/main/dist/telemetryReporter.d.ts#L34)